### PR TITLE
Removed hardcoded SVGs and replaced with react-icons

### PR DESCRIPTION
**Pull request by Sahil-101**
Hey I have removed hardcoded SVGs and replaced it with react-icons one taken from Bootstrap Icons(Search-icon) and Font Awesome (Heart-icon).

Bootstrap Icons : https://react-icons.github.io/react-icons/icons?name=bs
Font Awesome : https://react-icons.github.io/react-icons/icons?name=fa

### CODE REVIEW

**Review by ozer619 — APPROVED** [review]
Great work @Sahil-101 !! Everything looks good to me.

File: components/Navbar.js [hunk 1]
```diff
@@ -35,36 +38,12 @@ export default function Navbar() {
...
-            xmlns="http://www.w3.org/2000/svg"
-          >
-            <path
-              d="M4.318 6.318a4.5 4.5 0 000 6.364L12 20.364l7.682-7.682a4.5 4.5 0 00-6.364-6.364L12 7.636l-1.318-1.318a4.5 4.5 0 00-6.364 0z"
-              strokeLinecap="round"
-              strokeLinejoin="round"
-              strokeWidth="2"
-            />
-          </svg>
+          {<FaRegHeart className="inline-block w-5 h-5 stroke-current" />}
```
Comment (marekrozmus) [inline components/Navbar.js:41]: why the braces? `{` `}` ?

File: components/Navbar.js [hunk 2]
```diff
@@ -35,36 +38,12 @@ export default function Navbar() {
...
-            xmlns="http://www.w3.org/2000/svg"
-          >
-            <path
-              d="M21 21l-6-6m2-5a7 7 0 11-14 0 7 7 0 0114 0z"
-              strokeLinecap="round"
-              strokeLinejoin="round"
-              strokeWidth="2"
-            />
-          </svg>
+          {<BsSearch className="inline-block w-5 h-5 stroke-current" />}
```
Comment (marekrozmus) [inline components/Navbar.js:46]: same here

**Review by marekrozmus — CHANGES_REQUESTED** [review]

**Comment by marekrozmus** [comment]
Please add also to pull request description which issue it closes 🙂  - please read here https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue
If we add that information then when the PR is merged the linked issue will be closed automatically and there will be information in the issue which PR fixes it.

File: components/Navbar.js (hunk 2 above)
Comment (Sahil-101) [reply components/Navbar.js:46]: Oh Thanks, I will change this.

**Comment by audreyfeldroy** [comment]
Thanks for the PR @Sahil-101, and for addressing feedback so well! And I'm grateful for your code reviews here @marekrozmus and @ozer619.

**Comment by audreyfeldroy** [comment]
Tried it, looks good! Merging.

Thanks for linking the PR @RedFox0x20! I've added this to [Contributor Tips](https://margaritahumanitarian.gitbook.io/help-a-family-manual/contributor-tips) in the new Help-a-Family Manual.

**Pull request merged**